acme/src/acme/client.py

```diff
@@ -280,6 +282,12 @@ def poll_finalization(self, orderr: messages.OrderResource,
                     alt_chains = [self._post_as_get(url).text for url in alt_chains_urls]
                     orderr = orderr.update(alternative_fullchains_pem=alt_chains)
                 return orderr
+            elif body.status == messages.STATUS_PROCESSING:
```

Comment:
i personally like the pattern you established in the other if branches here of including an explanation of the different statuses, but please feel free to take or leave this suggestion. i don't think it's especially important

```suggestion
            elif body.status == messages.STATUS_PROCESSING:
                # "processing": The certificate is being issued.  Send a POST-as-GET request after
                # the time given in the Retry-After header field of the response, if any.
```